### ISSUE TITLE
Support for additive time signatures

### ISSUE THREAD

**Issue by bobderrico80** [0xfe/vexflow#987]
This is a feature request to support additive time signatures. For example, `.addTimeSignature('2+2+3/8')` would render a time signature like below:

![Screen Shot 2021-05-17 at 22 34 09](https://user-images.githubusercontent.com/7296842/118581697-0e530f80-b760-11eb-856f-e2db8698f41d.png)

**Comment by mscuthbert** [comment]
My sense from writing the parsing script in music21 for additive time signatures is that it's better to have a set of specialized functions that set a 2+2+3 numerator and one that sets a centered 8 denominator etc. than to try to parse 2+2+3/8, since there are definitely things like (2+3)/16 + 3/8 that get very hard to parse.

**Comment by rvilarl** [comment]
The request provided by @bobderrico80 to include additive time signatures would be quite simple if we had the standard glyph `timeSigPlus` in the fonts of the project. 
@0xfe where could I get that? would you like to include this feature?

@mscuthbert your example with alternating time signatures would require also `timeSigParensRight` & `timeSigParensLeft` the library already supports multiple calls to `addTimeSignature` so I believe that your example could then be:
`.addTimeSignature('(2+3)/16').addTimeSignature('+').addTimeSignature('3/8')`

**Comment by rvilarl** [comment]
@bobderrico80 can you look into #1242? Does that fullfil your request?
